pkg/controller/envoy/ads_event.go

```diff
@@ -160,6 +161,7 @@ func (svc *ServiceEvent) handleCdsResponse(rsp *service_discovery_v3.DiscoveryRe
 	// TODO: maybe we don't need to wait until all clusters ready before loading, like cluster delete
 
 	if len(svc.DynamicLoader.clusterNames) > 0 {
+		// Need to include all EDS, otherwise it will lead to being unsubscribed
```

Comment:
Didnot we want to compare clusterNames with last time?

Reply:
> Didnot we want to compare clusterNames with last time?

No comparison is required because this is a full push, which contains all cluster_eds pushed this time.

Reply:
Full push, doesnot mean we need to resend eds sub request when the cluster names are not changed